clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.kt

```
@@ -0,0 +1,991 @@
+                    val pushId = notif.optString(Constants.WZRK_PUSH_ID)
+                    val pushAccountId = notif.optString(Constants.WZRK_ACCT_ID_KEY)
+
+                    notifyListenerForPushImpressionSentToServer(
+                        PushNotificationUtil.buildPushNotificationRenderedListenerKey
+                            (
+                            pushAccountId,
+                            pushId
+                        )
+                    )
+                }
+            } catch (e: JSONException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)